website/templates/index.html

```diff
@@ -35,12 +35,10 @@ <h4 class="card-title">{% trans "Number Error" %}</h4>
                       </div>
                     </div>
                     <ul class="list-group list-group-flush">
-                        <li class="list-group-item">
-                            <a href="/search/?query=1&type=label">Examples</a>
-                        </li>
+                        <a class="btn btn-default" href="/search/?query=1&type=label">Examples</a>
                     </ul>   
                 </div>
-                <div class="col-sm-4">    
+                <div class="col-sm-3 cborder">
```

Comment:
Why are we changing from 4 to 3? Please read the bootstrap grids properly. Each row is divided into 12 sections (not 9), and hence the columns specified should add to 12.
Your changes will make the cards shift to left, leaving some space in the right making it inconsistent.